### PR TITLE
✨ Add request header displays

### LINKED ISSUE #53: dio看不到request header信息
## dio看不到request header信息

``` bash

  flutter_ume: ^0.3.0+1
  flutter_ume_kit_ui: ^0.3.0+1
  flutter_ume_kit_device: ^0.3.0
  flutter_ume_kit_perf: ^0.3.0
  flutter_ume_kit_show_code: ^0.3.0
  flutter_ume_kit_console: ^0.3.0
  flutter_ume_kit_dio: ^0.3.0
```

### PR DESCRIPTION
**Pull request by AlexV525**
Fix #53.

## Pull Request Checklist

- [x] I have read the [UME contribution document](../CONTRIBUTING.md) and understand how to contribute, commit the code according to the rules. 我已阅读过 UME 贡献文档，并了解如何进行贡献，按照规则提交了代码
- [x] I have added the necessary comments in code to ensure that other contributors can understand the reason for the change. 我在修改中已经添加了必要的注释，以确保便于其他贡献者理解修改原因
- [x] The code has been formatted by dartfmt before push. 代码在提交前已经经过 dartfmt 进行了格式化
- [x] Change does not involve the adjustment of test cases. Or all existing and new tests are passing. 改动不涉及测试用例调整，或 example 工程与单元测试已经完全跑通

### CODE REVIEW

File: kits/flutter_ume_kit_dio/lib/src/widgets/pluggable_state.dart [hunk 1]
```diff
@@ -229,6 +229,20 @@ class _ResponseCardState extends State<_ResponseCard> {
...
+  String? get _requestHeadersBuilder {
+    final Map<String, List<String>> map = _request.headers.map(
+      (key, value) => MapEntry(
+        key,
+        value is Iterable ? value.map((v) => v.toString()).toList() : [value],
+      ),
+    );
+    final Headers headers = Headers.fromMap(map);
+    if (!headers.isEmpty) {
+      return '$headers';
```
Comment (AlexV525) [inline kits/flutter_ume_kit_dio/lib/src/widgets/pluggable_state.dart]:
```suggestion
      return '\n$headers';
```

File: kits/flutter_ume_kit_dio/lib/src/widgets/pluggable_state.dart [hunk 2]
```diff
@@ -229,6 +229,20 @@ class _ResponseCardState extends State<_ResponseCard> {
...
+  String? get _requestHeadersBuilder {
+    final Map<String, List<String>> map = _request.headers.map(
+      (key, value) => MapEntry(
+        key,
+        value is Iterable ? value.map((v) => v.toString()).toList() : [value],
+      ),
+    );
+    final Headers headers = Headers.fromMap(map);
+    if (!headers.isEmpty) {
+      return '\n$headers';
```
Comment (talisk) [inline kits/flutter_ume_kit_dio/lib/src/widgets/pluggable_state.dart]: Do you think `_requestHeadersBuilder` should return a pure header `String` without the `\n`?

Comment (talisk) [reply kits/flutter_ume_kit_dio/lib/src/widgets/pluggable_state.dart]: And put it to the Widget tree

File: kits/flutter_ume_kit_dio/lib/src/widgets/pluggable_state.dart [hunk 3]
```diff
@@ -295,18 +309,24 @@ class _ResponseCardState extends State<_ResponseCard> {
...
-        return Container(
+        return Padding(
           padding: const EdgeInsets.symmetric(vertical: 8),
           child: Column(
             crossAxisAlignment: CrossAxisAlignment.start,
             children: <Widget>[
+              if (_requestHeadersBuilder != null)
+                _TagText(
+                  tag: 'Request headers',
+                  content: _requestHeadersBuilder!,
```
Comment (talisk) [inline kits/flutter_ume_kit_dio/lib/src/widgets/pluggable_state.dart]: Put the `\n` here.

**Review by talisk — COMMENTED** [review]
You may consider changing the `\n` again.

**Review by talisk — APPROVED** [review]
LGTM

**Pull request merged**